apis/v1alpha2/httproute_types.go

```diff
@@ -521,6 +523,15 @@ type HTTPRouteFilter struct {
 	// +optional
 	RequestRedirect *HTTPRequestRedirectFilter `json:"requestRedirect,omitempty"`
 
+	// URLRewrite defines a schema for a filter that responds to the
+	// request with an HTTP redirection.
+	//
+	// Support: Core
```

Comment:
The type `HTTPURLRewriteFilter` says it is Extended. Should this be extended?

Reply:
Updated, thanks!